src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.Core.verified.txt

```diff
@@ -3000,7 +3001,7 @@ namespace Akka.Event
         public Akka.Actor.IActorRef Sender { get; }
         public override string ToString() { }
     }
-    public class BusLogging : Akka.Event.LoggingAdapterBase
+    public sealed class BusLogging : Akka.Event.LoggingAdapterBase
```

Comment:
`sealed` `BusLogging`  and I'm considering making the constructor private in a subsequent PR so it can't be `new`'d up any longer - and will instead have to be populated by the `ActorSystem`. Considering doing that in order to enforce that configurable components like the `ILogMessageFormatter` and others are injected consistently across all call sites.